src/services/milestones/milestones.hooks.js

```diff
@@ -266,6 +323,7 @@ module.exports = {
     ],
     update: [restrict(), ...address, sanitizeHtml('description'), updatedAt],
```

Comment:
shouldn't we `checkEthConversion` for updates as well?

Reply:
yes we should!